app/main.py

```diff
@@ -1,7 +1,25 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+
+def get_weather(api_key: str) -> None:
+    city = "Paris"
+    base_url = f"http://api.weatherapi.com/v1/current.json?key=" f"{api_key}&q={city}"
+    response = requests.get(base_url)
+    weather = response.json()
```

Comment:
Recommendation: You should handle potential exceptions that can be thrown when making a request. For example, a requests.exceptions.RequestException could be thrown if there is a network problem, such as a DNS resolution failure, refused connection, etc. Consider using a try-except block to catch these exceptions and handle them appropriately.